question-service/src/main/resources/db/data/1.0/initial_schema.yaml

```diff
@@ -1,16 +1,16 @@
 databaseChangeLog:
-  - changeSet:
-      id: difficulty_enum_db_1.0
-      author: attila
-      changes:
-        - sqlFile:
-            dbms: postgresql
-            encoding: utf8
-            endDelimiter: \nGO
-            path: difficulty_enum.sql
-            relativeToChangelogFile: true
-            splitStatements: true
-            stripComments: true
+#  - changeSet:
```

Comment:
why did you com this

Reply:
we discouse that we can't define types in Postgress, at least not this case

Reply:
yes, i solved the problem. make a rebase